### ISSUE TITLE
Weird problem with v0.30.0+, while v0.29.0 is fine

### ISSUE THREAD

**Issue by andyzasl** [nerdswords/yet-another-cloudwatch-exporter#447]
Hi!
I'm using docker-compose mode, with ec2 instance role as credentials source.
After updating version to 0.30 (or 0.31) I've found, that it's not working.
I.e. 
curl localhost:5000/metrics - returns nothing.
docker logs yace 
shows only 2 lines:
```
{"level":"info","msg":"Parse config..","time":"2021-09-22T14:31:51Z"}
{"level":"info","msg":"Startup completed","time":"2021-09-22T14:31:51Z"}
```

Here is a config.yml 
```
apiVersion: v1alpha1
discovery:
  exportedTagsOnMetrics:
    ec2:
      - Name
      - Project
    ebs:
      - VolumeId
      - Project
    elb:
      - Project
    alb:
      - Project
      - Name
    rds:
      - Project
  jobs:
  - type: ec2
    period: 60
    length: 60
    regions:
      - us-east-1
      - eu-west-1
    metrics:
      - name: StatusCheckFailed
        statistics: [Maximum]
  - type: alb  # (AWS/ApplicationELB) - Application Load Balancer
    period: 600
    length: 600
    regions:
      - us-east-1
      - eu-west-1
    metrics:
      - name: UnHealthyHostCount
        statistics: [Maximum]
      - name: HealthyHostCount
        statistics: [Minimum]
  - type: elb
    regions:
      - us-east-1
      - eu-west-1
    length: 600
    period: 600
    metrics:
      - name: UnHealthyHostCount
        statistics: [Maximum]
      - name: HealthyHostCount
        statistics: [Minimum]
  - type: rds
    length: 600
    period: 600
    addCloudwatchTimestamp: false
    regions:
      - us-east-1
      - eu-west-1
    enableMetricData: false
    metrics:
          - name: CPUUtilization
            statistics: [Average]
          - name: DatabaseConnections
            statistics: [Average]
          - name: FreeStorageSpace
            statistics: [Average]
          - name: FreeableMemory
            statistics: [Average]
          - name: ReplicaLag
            statistics: [Maximum]
  - type: lambda
    regions:
      - us-east-1
      - eu-west-1
    enableMetricData: true
    period: 300
    length: 3600
    metrics:
     - name: Duration
       statistics: [Sum, maximum, Minimum, Average]
     - name: Invocations
       statistics: [Sum]
     - name: Errors
       statistics: [Sum]
     - name: Throttles
       statistics: [Sum]
     - name: DeadLetterErrors
       statistics: [Sum]
     - name: DestinationDeliveryFailures
       statistics: [Sum]
     - name: ProvisionedConcurrencyInvocations
       statistics: [Sum]
     - name: ProvisionedConcurrencySpilloverInvocations
       statistics: [Sum]
     - name: IteratorAge
       statistics: [Average, Maximum]
     - name: ConcurrentExecutions
       statistics: [Sum]
     - name: ProvisionedConcurrentExecutions
       statistics: [Sum]
     - name: ProvisionedConcurrencyUtilization
       statistics: [Maximum]
     - name: UnreservedConcurrentExecutions
       statistics: [Sum]
```

**Comment by DerFels** [comment]
For me it takes between 5 and 10 minutes before I can see metrics being scraped.
Did you just try and let it run for a while?

**Comment by andyzasl** [comment]
@DerFels yeah, I gave it about 10 minutes, still no results, no debug info, nothing.

**Comment by DerFels** [comment]
Mhh maybe the `-debug` parameter can help you?
See https://github.com/nerdswords/yet-another-cloudwatch-exporter/issues/446#issue-1002363943 for my command.
Otherwise I have no idea :)

**Comment by thomaspeitz** [comment]
@andyzasl could you please retry the newest version? - It should fix it.

**Comment by andyzasl** [comment]
Fixed with 0.31. Thanks!
